src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,213 @@
+          >
+            <img className={classes.icon} src={icon} alt={title} />
+            <Typography variant="body2">{title}</Typography>
+          </AccordionSummary>
+          <AccordionDetails className={classes.accordionDetail}>
+            <Grid container direction="column">
+              {layersCategories.map(
+                ({ title: categoryTitle, layers, tables }) => (
+                  <Grid
+                    item
+                    key={categoryTitle}
+                    className={classes.categoryContainer}
+                  >
+                    <Typography
+                      variant="body2"
+                      className={classes.categoryTitle}
+                    >
+                      {categoryTitle}
+                    </Typography>
+                    <hr />
+
+                    {layers.map(layer => {
+                      const { id: layerId, title: layerTitle } = layer;
+                      const selected = Boolean(
+                        selectedLayers.find(
+                          ({ id: testId }) => testId === layerId,
+                        ),
+                      );
+                      return (
+                        <div key={layerId} className={classes.layersContainer}>
+                          <Switch
+                            size="small"
+                            color="default"
+                            checked={selected}
+                            onChange={() => toggleLayerValue(selected, layer)}
+                            inputProps={{ 'aria-label': layerTitle }}
+                          />{' '}
+                          <Typography variant="body1">{layerTitle}</Typography>
+                        </div>
+                      );
+                    })}
+
+                    {tables.map(table => (
+                      <Button
+                        className={classes.button}
+                        id={table.title}
+                        key={table.title}
+                        onClick={() => showTableClicked(table)}
+                      >
+                        <Typography variant="body1">{table.title}</Typography>
+                      </Button>
+                    ))}
+                  </Grid>
+                ),
+              )}
+            </Grid>
+          </AccordionDetails>
+        </Accordion>
+      ))}
+    </>
+  );
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    button: {
+      textTransform: 'none',
+    },
+
+    title: {
+      margin: '0px 14px',
+      textTransform: 'uppercase',
+      color: 'inherit',
+
+      '&::after': {
+        content: '""',
+        display: 'inline-block',
+        width: 0,
+        height: 0,
+        marginLeft: 3.5,
+        verticalAlign: 3.5,
+        borderTop: '3.5px solid',
+        borderRight: '3.5px solid transparent',
+        borderBottom: 0,
+        borderLeft: '3.5px solid transparent',
```

Comment:
I am not sure this is the right approach. @Ry-DS do you have any recs?

Reply:
@ericboucher Sorry I forgot to remove that class, it was previously used by a button in the MenuItem before I replaced it using accordion. I updated it in the new commit 🙏